Contents/Code/constants.py

```diff
@@ -63,6 +63,14 @@
     show='TV Shows'
 )
 
+canonical_db = dict(
+    games='igdb',
+    game_collections='igdb',
+    game_franchises='igdb',
+    movies='themoviedb',
```

Comment:
'imdb' is also an option for movies... I think we need to handle that as well so the structure of this dict may not work.

![image](https://github.com/LizardByte/Themerr-plex/assets/42013603/6f5d3685-4b94-4b63-8727-3fda314e8098)

Reply:
Yep, however the all_page_X.json files give the themoviedb ID (this is what I call the "canonical" database)

So, even for movies with an IMDB ID, the cache is useless anyway since the list of IDs only contains one kind (themoviedb)

Reply:
But this code might be using the imdb db/id and not the themoviedb db/id

![image](https://github.com/LizardByte/Themerr-plex/assets/42013603/4e457092-922f-4400-bb0b-ba808a09f5b4)

And that will mean `item_may_exist` (as written now) will always be true for those cases.

```py
    if database != canonical_db[database_type]:
	        return True
```

Because `imdb` is never equal to `themoviedb`.

(This is the main reason for wanting to adjust the `item_may_exist` function.)

Reply:
fixed